spec.html

```diff
@@ -29884,7 +29884,7 @@ <h1>Properties of the String Prototype Object</h1>
         <h1>String.prototype.charAt ( _pos_ )</h1>
         <emu-note>
           <p>Returns a single element String containing the code unit at index _pos_ within the String value resulting from converting this object to a String. If there is no element at that index, the result is the empty String. The result is a String value, not a String object.</p>
-          <p>If `pos` is a value of Number type that is an integral Number, then the result of `x.charAt(pos)` is equal to the result of `x.substring(pos, pos + 1)`.</p>
+          <p>If `pos` is a value of Number type that is an integral Number, then the result of `x.charAt(pos)` is equivalent to the result of `x.substring(pos, pos + 1)`.</p>
```

Comment:
```suggestion
          <p>If `pos` is an integral Number, then `x.charAt(pos)` is equivalent to `x.substring(pos, pos + 1)`.</p>
```

As a side note, I have no idea what `x` is in this note. Do we need to introduce it with something like

> If `x` is a String and `pos` is an integral Number ...

Also, I think this note is only correct for "safe" integral Numbers. And only non-negative ones? I'm not sure about that last one.